includes/class-page-posts.php

```diff
@@ -89,7 +89,7 @@ public function output_posts() {
 		$wp_query = null;
 		$wp_query = $temp;
 		wp_reset_query();
-		remove_filter( 'excerpt_more', array( &$this, 'custom_excerpt_more' ) );
+		remove_filter( 'excerpt_more', array( $this, 'custom_excerpt_more' ) );
```

Comment:
@bradyvercher Is the pass by reference `&$this` removed because of the [reason stated here](https://stackoverflow.com/a/27077115) - it's not needed for objects because they're passed by default?

Reply:
Yeah, objects are passed by reference by default in PHP 5+. They were passed by value in PHP 4, which I believe is where that syntax originated, but I haven't seen it in a long time. Here's an article with some more information: https://tomjn.com/2014/01/08/this/